modules/google-analytics/classes/wp-google-analytics-legacy.php

```diff
@@ -69,6 +69,13 @@ public function insert_code() {
 			return;
 		}
 
+		if ( class_exists( 'Jetpack_AMP_Support' )
```

Comment:
I've always wondered about this `class_exists` check. Do you think it's necessary? The class should always exist as it's loaded through the `load-jetpack.php` and `3rd-party.php` files before Jetpack is initiated.

Reply:
It used to be necessary a few months back, when that class didn't exist yet in the WordPress.com environment because the AMP plugin was not up to date there.

Now that WordPress.com is up to date and the class always loaded there, it isn't really as necessary.

Reply:
Thank you both on this one.